src/game/savegame/savegame.c

```diff
@@ -614,3 +616,30 @@ bool Savegame_RestartAvailable(int32_t slot_num)
     SAVEGAME_INFO *savegame_info = &m_SavegameInfo[slot_num];
     return savegame_info->features.restart;
 }
+
+GAMEFLOW_OPTION Savegame_PlayAvailableStory(int32_t slot_num)
+{
+    SAVEGAME_INFO *savegame_info = &m_SavegameInfo[slot_num];
+
+    int32_t gf_option = GF_START_GAME | g_GameFlow.first_level_num;
+
+    bool loop_continue = true;
+    while (loop_continue) {
+        int32_t gf_direction = gf_option & ~((1 << 6) - 1);
+        int32_t gf_param = gf_option & ((1 << 6) - 1);
+
+        gf_option =
+            GameFlow_StorySoFar(gf_param, GFL_NORMAL, savegame_info->level_num);
```

Comment:
Unnecessary assignment

Reply:
See above about return value. I think it is necessary.